libchannelserver/ChannelRPCServer.cpp

```diff
@@ -846,15 +846,16 @@ std::vector<dev::channel::ChannelSession::Ptr> ChannelRPCServer::getSessionByTop
     std::vector<dev::channel::ChannelSession::Ptr> activedSessions;
 
     std::lock_guard<std::mutex> lock(_sessionMutex);
-    for (auto& it : _sessions)
+    for (auto it : _sessions)
```

Comment:
why can not use & ?

Reply:
fixed in: https://github.com/FISCO-BCOS/FISCO-BCOS/pull/527/files#diff-8bf794f0b9e4c11b691b052256f5afa4